index.js

```diff
@@ -0,0 +1,61 @@
+const db = require('./data/db');
+const express = require('express');
+const bodyParser = require('body-parser')
+
+const server = express();
+server.use(bodyParser.json());
```

Comment:
Express now includes body-parser middleware by default! So just use `server.use(express.json());` in lieu of bodyParser.